packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/javascript/guide.md

```diff
@@ -1,3 +1,406 @@
+          location.reload();
+        }
+      </script>
+    </body>
+  </html>
+  ```
+
+3. Configure the code in `index.html` with values for your Okta org application integration:
+
+    * **baseUrl:** `"https://${yourOktaDomain}"`. For example, `"http://example.okta.com"`
+    * **redirectUri:** `"https://${yourAppRedirectUri}"`. For example, `"http://localhost:3000"`
+    * **clientId:** `"${yourClientId}"`. For example, `0oa2am3kk1CraJ8xO1d7`
+    * **issuer:** `"https://${yourOktaDomain}/oauth2/default"`. For example, `"https://example.okta.com/oauth2/default"`
+
+4. (Optional) Update the version of the `okta-auth-js` dependency to make use of other authentication features in your sample directory, `simple-spa`. See [Related SDKs](https://github.com/okta/okta-signin-widget#related-sdks). The basic authentication feature does not require this update.
+
+    ```bash
+    npm install @okta/okta-auth-js
+    ```
+
+    >**Note:** The SPA app logout function requires `okta-auth-js` 5.10 or greater.
+
+#### Run the sample application
+
+1. In your sample directory, `simple-spa`, you can run a static site web server using the following command:
+
+    ```bash
+    npx http-server . -p 3000
+    ```
+
+    >**Note:** If not installed previously, you will be prompted to install the `npx` package.
+
+1. Navigate to the local page, `http://localhost:3000`. The simple web page appears with a message that you're not signed in and displays the Sign-In Widget.
+
+1. Sign in with a user from your org assigned to the app integration. The simple web page appears with the signed-in user's email address.
+
+### Sign In to Okta with the Default Dashboard
+
+In this case, you use the Widget to sign in to the default Okta dashboard. This requires taking the Widget initialization code, and modifying the success behavior to redirect to your org's dashboard.
+
+```javascript
+function success(res) {
+  if (res.status === 'SUCCESS') {
+    res.session.setCookieAndRedirect('https://${yourOktaDomain}/app/UserHome');
+  }
+}
+```
+
+### Sign In to Okta and SSO Directly to an App
+
+If you want to sign the user directly in to an application within Okta, you redirect to the specific URL for that application. To find that URL, go to that application's page in your Okta org and find [the embed link](https://help.okta.com/okta_help.htm?id=ext-apps-page-show-application-embed-links).
+
+### Sign In to Okta with a Custom Dashboard
+
+If you are signing your users in to Okta, but you don't want to use the Okta dashboard, then you can change the redirect URL to point to your custom portal instead.
+
+```javascript
+function success(res) {
+  if (res.status === 'SUCCESS') {
+    res.session.setCookieAndRedirect('https://example.com/dashboard');
+  }
+}
+```
+
+### Sign In to Your Application
+
+If want to use the Widget to sign in to your own application instead of Okta, you need to [set-up a custom Authorization Server](/docs/guides/customize-authz-server/) in Okta.
```

Comment:
```suggestion
If want to use the Widget to sign in to your own application instead of Okta, you need to [set up a Custom Authorization Server](/docs/guides/customize-authz-server/) in Okta.
```